server/src/lib/model/db/migrations/20191015151914-add-teams-and-user-client-teams.ts

```diff
@@ -0,0 +1,28 @@
+export const up = async function(db: any): Promise<any> {
+  return db.runSql(
+    `
+    CREATE TABLE teams (
+      team_id CHAR(36) NOT NULL PRIMARY KEY,
+      teamname TEXT NOT NULL DEFAULT (''),
+      leader CHAR(36) NOT NULL,
+      invite_link TEXT DEFAULT NULL DEFAULT (''),
+      created_at DATETIME DEFAULT now(),
+      logo_url TEXT DEFAULT NULL DEFAULT (''),
+      FOREIGN KEY (leader) REFERENCES user_clients(client_id)
+    );
+    
+    CREATE TABLE user_client_teams (
+      client_id CHAR(36) NOT NULL,
+      team_id CHAR(36) NOT NULL,
+      created_at DATETIME DEFAULT now(),
+      PRIMARY KEY (client_id, team_id),
+      FOREIGN KEY (client_id) REFERENCES user_clients(client_id),
+      FOREIGN KEY (team_id) REFERENCES teams(team_id)
+    );
```

Comment:
`teams` to `user_clients` is a one-to-many relationship, so we could also do this with a foreign key on the user. Sorry if I didn't make that clear! Did you store a junction table like this for safety / efficiency, or do you think a foreign key would be better?

Reply:
sorry, I didn't realize it is a one-to-many relationship. My first thought was that maybe in the future, people could join different teams or create multiple teams (and be a role like leader to manage the teams, like changing the logo etc.).